### DISCUSSION TITLE
Control inverter from Home Assistant

### DISCUSSION THREAD

**Discussion by MarkiyanSHCH** [softwarecrash/Solar2MQTT-ESP8266#120] [General]
Hi guys!

Has someone already implemented yaml file for HA to control all inverter parameters?

**Comment by messageexplorer** [comment]
What reason is for? As for me one parameter "Max utility charging current" can be changed. But this can be achived by sending MQTT command from HA.

**Reply by MarkiyanSHCH** [reply]
Just ask, maybe someone already did it

**Reply by softwarecrash** [reply]
you can control all data with custom commands, for the commands search in the protocol files

**Reply by MarkiyanSHCH** [reply]
I know it
